client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -135,6 +135,69 @@ function mediaButton( editor ) {
 		return { isLoaded, onLoad };
 	} )();
 
+	function parseShortcode( content ) {
```

Comment:
I copied this function from `tinymce/plugins/wpeditimage/plugin`. We should move it to a shared helper file. However, it depends on `tinymce` so might be large.